tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -67,6 +68,22 @@ def _make_virtwho_configure(self):
             args[u'hypervisor_password'] = self.hypervisor_password
         return args
 
+    def _try_to_get_guest_bonus(self, hypervisor_name, sku):
+        subscriptions = entities.Subscription().search(
+            query={'search': '{0}'.format(sku)})
+        for item in subscriptions:
+            item = item.read_json()
+            if hypervisor_name.lower() in item['hypervisor']['name']:
+                return item['id']
+
+    def _get_guest_bonus(self, hypervisor_name, sku):
+        vdc_id, time = wait_for(self._try_to_get_guest_bonus,
+                                func_args=(hypervisor_name, sku),
+                                fail_condition=None,
+                                timeout=5,
```

Comment:
Is it enough to set `timeout=5`, should we wait for a long time if there is no bonus generated? such as 15s?

Reply:
ok, I will give it longer